### ISSUE TITLE
Questions about implementation

### ISSUE THREAD

**Issue by shimon-vainer** [huanngzh/EpiDiff#1]
Hi dear Authors! 

If possible, I would like to inquire about certain parts which are not fully clear from the paper:

- where in the UNet the ECA blocks are inserted? (between self-attention and cross-attention, or after the cross-attention?)
- does the ray self-attention output is added as a residual to the prior block output in the unet? 
- what are the hyperparameters for the ECA block? number of heads, dimensions, etc.
- does the weighted sum for the ray self-attention comes from summing directly on the values in the sample dimension after the softmax?
- what are the hyperparameters for the positional encoding for the "harmonic transformation"? how many frequencies?

thank you!

**Comment by huanngzh** [comment]
Below is a reply regarding implementation details.

- The ECA blocks are inserted after the cross-attention.
- The ray self-attention is **not** added as a residual to the prior block output. We have tried adding it as a residual to the original feature, but the experimental results are not as good as not using it as a residual.
- Refer to [feature_aggregator.py](https://github.com/huanngzh/EpiDiff/blob/dev/epidiff/models/extras/feature_aggregator.py).
- Refer to [utils/project.py](https://github.com/huanngzh/EpiDiff/blob/dev/epidiff/utils/project.py#L313).

In addition, we released the training code of our minimal baseline in the `dev` branch. We will gradually release the complete code and model.
